.gitmodules

```diff
@@ -86,3 +86,7 @@
 	path = sources/meta-elm-binary
 	url = git://github.com/yoedistro/meta-elm-binary
 	branch = master
+[submodule "sources/meta-rockchip"]
+	path = sources/meta-rockchip
+	url = git://git.yoctoproject.org/meta-rockchip
+	branch = dunfell
```

Comment:
should this be the master branch in meta-rockchip?

Reply:
yes it should be `master`

Reply:
Thanks @cbrake @kraj, I will fix this.